ui/app/home/controllers/loginController.js

```diff
@@ -71,8 +71,9 @@ angular.module('bahmni.home')
                 });
             });
 
-            $scope.isChrome = function () {
-                if ($window.navigator.userAgent.indexOf("Chrome") != -1) {
+            $scope.isChromeOrFirefox = function () {
```

Comment:
would be better to keep the function names agnostic of the impl... so `isSupportedBrowser`